_data/locales/en.yml

```diff
@@ -401,8 +401,8 @@ leap_ovw_desc: Leap uses source from SUSE Linux Enterprise (SLE), which gives Le
 tumbleweed_ovw_title: State-of-the-art desktop and server operating system
 tumbleweed_ovw_desc: With Tumbleweed you don't have to take difficult decisions about things you value, either freedom or safety, either control or security, technology or stability -- Tumbleweed lets you have your cake and eat it too!
 leap_micro_ovw_title: Leap Micro is an ultra-reliable, lightweight operating system built for containerized and virtualized workloads.
-leap_micro_ovw_desc_p1: This community version is based on SUSE Linux Enterprise Micro, which leverages the enterprise hardened security and compliance components of SUSE Linux Enterprise. This merging of technologies provides for a modern, immutable and developer-friendly OS platform.
-leap_micro_ovw_desc_p2: Leap Micro offers an offline image. The main difference between the offline and self-install/raw images are that the offline image has an installer. Raw and self-install allows for customization via combustion or manually in the image after it is written to the disk. There is an option for a real-time kernel.
+leap_micro_ovw_desc_p1: This community version is based on SUSE Linux Micro, which leverages the enterprise-hardened security and compliance components of SUSE Linux Enterprise. This merging of technologies provides for a modern, immutable, and developer-friendly OS platform.
+leap_micro_ovw_desc_p2: Leap Micro since 6.0 comes with the self-install image, raw image, and a Packaged DVD that can serve as an offline repository. Self-install image is our recommendation for any USB-based installation since we no longer use traditional installers. Users can automate customization with either <a href="https://opensuse.github.io/fuel-ignition/">ignition</a> or <a href="https://github.com/openSUSE/combustion">combustion</a> or use the jeos-firstboot wizard.
```

Comment:
I think "offers" is fitting better than "comes with" since you do not automatically get all these things with your installation.

Plus some minor nitpicks:

> _a_ raw image

> a _packaged_ DVD

> _The_ self-install image ...

> _Users can automate and customize their installation_ with either ... _Ignition_ or _Combustion_ ... or _using_ the ...

Reply:
Makes sense I'll change it back to offers.

Reply:
The filename of DVD is called "Packages DVD"